src/main/java/in/ac/bits/protocolanalyzer/analyzer/AnalyzerCell.java

```diff
@@ -108,6 +108,7 @@ public void addCustomAnalyzer(CustomAnalyzer analyzer,
 	 */
 	public boolean takePacket(PacketWrapper packet) {
 		try {
+			//System.out.println("Packet offered to the queue by takePacket() in AnalyzerCell = " + packet );
```

Comment:
@adityaketkar  There are a few codeclimate issues. Please remove them except for the ones related to comments. You can retain the comments.